### PR TITLE
LPS-154576 buildLang

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.json [hunk 1]
```diff
@@ -5,8 +5,7 @@
 				{
 					"dataType": "int",
 					"defaultValue": "4",
-					"description": "number-of-tabs",
-					"label": "Number Of Tabs",
+					"label": "number-of-tabs",
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.json:8]: :0 Que desastre esto

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/CheckboxField.js [hunk 2]
```diff
@@ -91,6 +95,12 @@ export function CheckboxField({disabled, field, onValueSelect, title, value}) {
 						/>
 					)}
 			</div>
+
+			{field.description ? (
+				<div className="mt-1 small text-secondary" id={helpTextId}>
+					{field.description}
+				</div>
+			) : null}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-configuration-fields/CheckboxField.js:103]: Very nice 😍

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.json [hunk 3]
```diff
@@ -17,6 +17,18 @@
 				}
 			],
 			"label": "tabs-options"
+		},
+		{
+			"configurationRole": "advanced",
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/tabs/index.json:22]: Do we need this to be advanced? It is a fairly simple option

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:reevaluate:1397401_100

**Comment by ealonso** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/8238

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**